src/pybind/rados/rados.pyx

```diff
@@ -1356,7 +1287,11 @@ Rados object in state %s." % self.state)
         io.io = ioctx
         return io
 
-    def mon_command(self, cmd, inbuf, timeout=0, target=None):
+    def mon_command(self,
+                    cmd: str,
+                    inbuf: bytes,
```

Comment:
FYI -- this change is breaking OpenStack. The Glance driver has for the last 4+ years been passing an empty string here but now these type annotations are causing a failure [1] since now it's enforcing that this must be a bytes object.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1923789